src/network/protocol/messages/merkleblock.zig

```diff
@@ -0,0 +1,146 @@
+const std = @import("std");
+const protocol = @import("../lib.zig");
+
+const Sha256 = std.crypto.hash.sha2.Sha256;
+const BlockHeader = protocol.BlockHeader;
+const CompactSizeUint = @import("bitcoin-primitives").types.CompatSizeUint;
+
+/// MerkleBlockMessage represents the "MerkleBlock" message
+///
+/// https://developer.bitcoin.org/reference/p2p_networking.html#merkleblock
+pub const MerkleBlockMessage = struct {
+    block_header: BlockHeader,
+    transaction_count: u32,
+    hash_count: CompactSizeUint,
+    hashes: []const [32]u8,
+    flag_bytes: CompactSizeUint,
+    flags: []const u8,
+
+    const Self = @This();
+
+    pub inline fn name() *const [12]u8 {
```

Comment:
after reading the doc it appears that it is better to remove inline, I did it for the others in #131